content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -413,28 +413,43 @@ When you don't need to attach additional information to the relation, you can mo
 
 ## MongoDB
 
-In MongoDB, m-n-relations are represented by:
+In MongoDB, many-to-many relations are represented by:
 
-- Relation fields on both sides, include a `@relation` attribute on _each side_
-- A scalar list of referenced IDs on each side, including the `@db.ObjectId` attribute
+- a `@relation` attribute on _each side_, with mandatory `fields` and `references` arguments
+- a scalar list of referenced IDs on each side, including the `@db.ObjectId` attribute
```

Comment:
Does the list really have to be `@db.ObjectId`? Is that not only true if the referenced field is an ObjectId as well, but it could very well also be an Int?

Reply:
yes it can be something else, indeed. The only requirement is that the type of the scalar list field on one side matches the id field type on the other side.

Reply:
Thanks, I've updated this, though may need phrasing better.